infrastructure/ami/scripts/install-huggingface-libraries.sh

```diff
@@ -9,24 +9,22 @@ echo "TRANSFORMERS_VERSION: $TRANSFORMERS_VERSION"
 echo "OPTIMUM_VERSION: $OPTIMUM_VERSION"
 
 pip install --upgrade --no-cache-dir \
-    "transformers[sklearn,sentencepiece,vision]==$TRANSFORMERS_VERSION" \
-    "datasets==2.16.1" \
-    "accelerate==0.23.0" \
-    "diffusers==0.25.0" \
```

Comment:
are those all controlled by optimum?

Reply:
All those are now controlled by extras of optimum-neuron, hard-coded this way could lead to dependencies conflicts and it's a bit difficult to maintain.

Reply:
👍🏻